test1-dev.js

```diff
@@ -3,3 +3,9 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var fooAll = '' ;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-test-dev/pullRequest?prid=1986)
```suggestion
var fooAll = "" ;
```